### ISSUE TITLE
null object error in onActivityResult

### ISSUE THREAD

**Issue by christinachimi** [BlinkID/blinkid-flutter#11]
Seeing errors like:

```
com.microblink.flutter.MicroblinkFlutterPlugin.onActivityResult
MicroblinkFlutterPlugin.java, line 171
java.lang.NullPointerException: Attempt to invoke interface method 'void io.flutter.plugin.common.MethodChannel$Result.success(java.lang.Object)' on a null object reference
```

Stack trace at the bottom. It looks like you might be addressing this in the next release. Is there a timeline on getting that out?

We are mostly seeing this with live users, still working on recreating it.

App Center provides two stack traces:

```
com.microblink.flutter.MicroblinkFlutterPlugin.onActivityResult MicroblinkFlutterPlugin.java:171
io.flutter.embedding.engine.FlutterEnginePluginRegistry$FlutterEngineActivityPluginBinding.onActivityResult FlutterEnginePluginRegistry.java:691
io.flutter.embedding.engine.FlutterEnginePluginRegistry.onActivityResult FlutterEnginePluginRegistry.java:378
io.flutter.embedding.android.FlutterActivityAndFragmentDelegate.onActivityResult FlutterActivityAndFragmentDelegate.java:619
io.flutter.embedding.android.FlutterActivity.onActivityResult FlutterActivity.java:584
android.app.Activity.dispatchActivityResult Activity.java:8135
android.app.ActivityThread.deliverResults ActivityThread.java:4838
android.app.ActivityThread.handleSendResult ActivityThread.java:4886
android.app.servertransaction.ActivityResultItem.execute ActivityResultItem.java:51
android.app.servertransaction.TransactionExecutor.executeCallbacks TransactionExecutor.java:135
android.app.servertransaction.TransactionExecutor.execute TransactionExecutor.java:95
android.app.ActivityThread$H.handleMessage ActivityThread.java:2016
android.os.Handler.dispatchMessage Handler.java:107
android.os.Looper.loop Looper.java:214
android.app.ActivityThread.main ActivityThread.java:7356
java.lang.reflect.Method.invoke Method.java
com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run RuntimeInit.java:492
com.android.internal.os.ZygoteInit.main ZygoteInit.java:930
```

```
android.app.ActivityThread.deliverResults ActivityThread.java:4845
android.app.ActivityThread.handleSendResult ActivityThread.java:4886
android.app.servertransaction.ActivityResultItem.execute ActivityResultItem.java:51
android.app.servertransaction.TransactionExecutor.executeCallbacks TransactionExecutor.java:135
android.app.servertransaction.TransactionExecutor.execute TransactionExecutor.java:95
android.app.ActivityThread$H.handleMessage ActivityThread.java:2016
android.os.Handler.dispatchMessage Handler.java:107
android.os.Looper.loop Looper.java:214
android.app.ActivityThread.main ActivityThread.java:7356
java.lang.reflect.Method.invoke Method.java
com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run RuntimeInit.java:492
com.android.internal.os.ZygoteInit.main ZygoteInit.java:930
```

**Comment by matvidako** [comment]
Hi,

planned release date for version 5.8 including this fix is 30th September.

**Comment by anjapenic** [comment]
Hello @christinachimi,

Since this was fixed in v5.8.0, we're closing this issue. If you have any further questions or feedback, feel free to contact us at support@microblink.com. 

Best regards,
Anja
